students/12th/juheejang/login.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <div class="wrapLogin">
+      <div class="wrapLoginMain">
+        <div class="logoImage">
+          <img src="img/logo_text.png" alt="instagram logo" width="175" height="51">
+        </div>
+        <form>
+          <input type="text" id="userAcount" name="userAcount" placeholder="전화번호, 사용자 이름 또는 이메일" style="width: 258px; height: 36px;">
+          <br>
+          <input type="password" id="userPassword" name="userPassword" placeholder="비밀번호" style="width: 258px; height: 36px;">
+          <br>
+          <input type="button" id="loginButton" value="로그인">  
+        </form>
+        <div>
+          <hr>
+          <p><span class="facebookLogo"></span><span>Facebook으로 로그인</span></p>
```

Comment:
```suggestion
          <p>
            <span class="facebookLogo"></span>
            <span>Facebook으로 로그인</span>
          </p>
```
부모 자식 관계를 잘 드러낼 수 있도록 줄바꿈 해주세요!

Reply:
확인하였습니다 😀